content/millennium/r4/conformance/structure-definition.md

```diff
@@ -75,6 +75,7 @@ Authorization is not required.
  [`account-related-parts`]                                   | A reference to other related Accounts.
  [`account-state`]                                           | The status of the Account within the billing or correspondence workflow.
  [`client-organization`]                                     | The financially responsible organization.
+ [`clinical-instruction`]                                    | The clinical instructions to display the medication order.
```

Comment:
I thought we had the definition something like this "The clinical instructions for an order that are intended for healthcare providers."

Reply:
updated here : https://github.com/cerner/fhir.cerner.com/commit/d19ed6fbc1011e62ffd2869eff9f1ed6975cdd0d#diff-f10dc1eb3735aa1264e82ae17b8a0cd967fc69912568786cd6406be319d3fd17R78